src/test/resources/application.yml

```diff
@@ -17,3 +17,9 @@ spring:
     hibernate:
       naming:
         physical-strategy: org.hibernate.boot.model.naming.PhysicalNamingStrategyStandardImpl
+
+ays:
+  scheduler:
+    UnusedInvalidTokenDeletionScheduler:
+      cron: '0/1 * * ? * *'
+      enabled: true
```

Comment:
Parametreleri burada yönetmek zorunda değiliz, değişken olabilecek bir yapı değil

Reply:
Test ortamında daha kısa bir vakit verebilmek için kullanılıyor.